### PR TITLE
che #10444: Fixing NumberFormatException occurences during the 'watchEvents' calls on k8s infrastructure

### PR DESCRIPTION
**Pull request by ibuziuk**
### What does this PR do?
Fixing NumberFormatException occurences during the 'watchEvents' calls on k8s infrastructure

### What issues does this PR fix or reference?
#10444 

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/util/PodEvents.java [hunk 1]
```diff
@@ -30,6 +31,9 @@ private PodEvents() {}
    * Converts the time of {@link PodEvent} e.g. '2018-05-15T16:17:54Z' to the {@link Date} format
    */
   public static Date convertEventTimestampToDate(String timestamp) throws ParseException {
+    if (StringUtils.isBlank(timestamp)) {
```
Comment (skabashnyuk) [inline infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/util/PodEvents.java]: can you use 
org.eclipse.che.ide.util.StringUtils.isNullOrEmpty or
com.google.common.base.Strings.isNullOrEmpty ?

Comment (ibuziuk) [reply infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/util/PodEvents.java]: yeah, sure

**Comment by ibuziuk** [comment]
@skabashnyuk btw, I have added a test for `12.` timestamp which for some reason also throw NFE according to the logs, but I do not really understand how did it happen ? According to the test ParsingException should be thrown

[1] https://github.com/eclipse/che/pull/10546/files#diff-8cf067455ba8932931a2162e0e1228fdR39

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/util/PodEvents.java (hunk 1 above)
Comment (garagatyi) [reply infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/util/PodEvents.java]: +1

**Review by garagatyi — APPROVED** [review]
LGTM, but please, replace usage of a utility method to avoid adding a new dependency to the module.

**Comment by ibuziuk** [comment]
PR has been updated

**Comment by ibuziuk** [comment]
[ci-test]

**Pull request merged**